context.js

```diff
@@ -437,6 +440,15 @@ function destroyNamespace(name) {
   assert.ok(namespace, 'can\'t delete nonexistent namespace! "' + name + '"');
   assert.ok(namespace.id, 'don\'t assign to process.namespaces directly! ' + util.inspect(namespace));
 
+  namespace._hook.disable();
+
+  /**
+   * Zerofying _contexts as haviest part of Namespace
+   * in case our namespace is retained mistakenly, so
+   * we releasing heavist part at least
+   */
+  namespace._contexts = null;
```

Comment:
Couple of typos, and this shouldn't be JSDoc block. Maybe:
```
/*
 * Zeroing _contexts as heaviest part of Namespace
 * In case our namespace is retained mistakenly, so
 * at least we are releasing heaviest part
 */
```

Reply:
Yeah, I agree with these changes. Thanks for suggestions.